### PR TITLE
Initial Plum Lightpad component documentation

### PR DESCRIPTION
**Pull request by ColinHarrington**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant/pull/16576):** home-assistant/home-assistant#16576

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. 
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/plum_lightpad.markdown [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+comments: false
+sharing: true
+footer: true
+ha_category: Hub
+ha_iot_class: "Local Push"
+featured: true
+ha_release: "0.80"
+---
+
+Plum Lightpad
```
Comment (frenck) [inline source/_components/plum_lightpad.markdown]: This is a... duplicate title?
I suggest removing it.

File: source/_components/plum_lightpad.markdown [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+footer: true
+ha_category: Hub
+ha_iot_class: "Local Push"
+featured: true
+ha_release: "0.80"
+---
+
+Plum Lightpad 
+
+"The most advanced smart dimmer. Ever."  https://plumlife.com/
```
Comment (frenck) [inline source/_components/plum_lightpad.markdown]: Maybe make a real link out of it?

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks, @ColinHarrington!

Just a couple of tiny comments. Could you take a look? 👍

File: source/_components/plum_lightpad.markdown [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+description: "Instructions on setting up Plum Lightpads within Home Assistant."
+date: 2018-10-15 17:58
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Hub
+ha_iot_class: "Local Push"
+featured: true
+ha_release: "0.80"
```
Comment (tjorim) [inline source/_components/plum_lightpad.markdown]: 0.80 is the current release, not the next one.

File: source/_components/plum_lightpad.markdown [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+To enable Plum Lightpad support, add the following lines to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+plum_lightpad:
+  username: YOUR_CLOUD_USERNAME
+  password: YOUR_CLOUD_PASSWORD
+```
+
+{% configuration %}
```
Comment (tjorim) [inline source/_components/plum_lightpad.markdown:42]: Remove 2 leading spaces from each line in this section to fix indentation.

**Review by tjorim — CHANGES_REQUESTED** [review]
Here are some more, nice work on the component by the way!

File: source/_components/plum_lightpad.markdown (hunk 3 above)
Comment (ColinHarrington) [reply source/_components/plum_lightpad.markdown]: I wasn't sure, what should I put here?  It's an easy update for whenever it goes in

Comment (tjorim) [reply source/_components/plum_lightpad.markdown]: For now it's ok, just a reminder to update when it's done.

**Comment by ColinHarrington** [comment]
Link to the component page: https://deploy-preview-6837--home-assistant-docs.netlify.com/components/plum_lightpad/

**Comment by ColinHarrington** [comment]
I also asked the company if we could use their logo in our site https://twitter.com/ColinHarrington/status/1052019251798781952

Is there a standard for incorporating a logo into the documentation?

File: source/_components/plum_lightpad.markdown (hunk 4 above)
Comment (ColinHarrington) [reply source/_components/plum_lightpad.markdown:42]: Done

**Comment by frenck** [comment]
There is not a real standard. Take a look at the other docs for an example.

**Comment by tjorim** [comment]
Not stale, parent-pr has been merged already.

**Review by frenck — APPROVED** [review]

File: source/_components/plum_lightpad.markdown [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+title: "Plum Lightpad"
+description: "Instructions on setting up Plum Lightpads within Home Assistant."
+date: 2018-10-15 17:58
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Switch
+ha_iot_class: "Local Push"
+featured: true
```
Comment (balloob) [inline source/_components/plum_lightpad.markdown:12]: We should be more careful, this should not have been featured.

**Pull request merged**